module-2/backend/week_three.md

```diff
@@ -10,14 +10,56 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+
+* rails new
```

Comment:
rails new -T --skip-spring --skip-turbolinks (this is more common than just rails new)